src/Language/PureScript/Ide/Rebuild.hs

```diff
@@ -51,12 +50,11 @@ rebuildFile path cacheSuccess = do
 
   input <- liftIO $ readUTF8File path
 
-  m <- P.addDefaultImport (P.ModuleName [P.ProperName "Prim"]) <$>
-    case map snd <$> P.parseModulesFromFiles id [(path, input)] of
-      Left parseError ->
-        throwError . RebuildError . toJSONErrors False P.Error $ parseError
-      Right [m] -> pure m
-      Right _ -> throwError . GeneralError $ "Please define exactly one module."
+  m <- case map snd <$> P.parseModulesFromFiles id [(path, input)] of
+    Left parseError ->
+      throwError . RebuildError . toJSONErrors False P.Error $ parseError
+    Right [m] -> pure m
+    Right _ -> throwError . GeneralError $ "Please define exactly one module."
```

Comment:
I think this case should be an `internalError` now, since we _only_ allow a single module per file.